code/game/machinery/doors/shutters.dm

```diff
@@ -10,6 +10,9 @@
 	max_integrity = 100
 	recipe_type = /datum/crafting_recipe/shutters
 
+/obj/machinery/door/poddoor/shutters/add_context(atom/source, list/context, obj/item/held_item, mob/user)
+	return NONE
+
```

Comment:
don't do this, make it so that the /door subtype, have a variable that determines if the door can be opened by hand or not. Then implement this into the add_context override of the /door subtype and set it to FALSE on poddoors